Resources/translations/default.fr.yml

```diff
@@ -52,6 +52,11 @@ layout_model:
     list: Liste des modèles
     uploaded: Modèle enregistré
 
```

Comment:
Suggestio in 'global', as global words :
    landscape: Paysage
    portrait: Portrait
    undefined: Non défini